src/main/java/eight/exception/NoSuchGenreException.java

```diff
@@ -0,0 +1,7 @@
+package eight.exception;
+
+public class NoSuchGenreException extends Exception {
```

Comment:
As discussed previously better approach is to extend from some kind of RuntimeException

Reply:
Fixed